demos/next/README.md

```diff
@@ -0,0 +1,23 @@
+# useAckee with Next.js
```

Comment:
README's all have the same structure, adapted from the one made by `create-next-app`.